pages/patterns/create-standard.js

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+import { Theme } from '@twilio-paste/core/theme';
+import { Box } from '@twilio-paste/core/box';
+import { Button } from '@twilio-paste/core/button';
+import { Heading } from '@twilio-paste/core/heading';
+import { Paragraph } from '@twilio-paste/core/paragraph';
+import { Stack } from '@twilio-paste/core/stack';
+import { useToaster, Toaster } from '@twilio-paste/core/toast';
+
+import { CreateServiceModal } from '../../components/site/patterns/Create Patterns/CreateServiceModal';
+import { ServicesTable } from '../../components/site/patterns/Create Patterns/ServicesTable';
+import { serviceData } from '../../components/site/patterns/Create Patterns/data';
+
+export default function Create() {
+  // Table state
+  const [services, setServices] = React.useState(serviceData);
+  // Modal state
+  const [isModalOpen, setModalOpen] = React.useState(false);
+  const openModal = () => setModalOpen(true);
+  const closeModal = () => setModalOpen(false);
+  // Toaster properties
+  const toaster = useToaster();
+
+  const handleFormSubmit = ({ serviceName, serviceSID }) => {
+    // Update our table state
+    setServices([
+      {
+        name: serviceName,
+        sid: serviceSID,
+        created: new Date().toString(),
+      },
+      ...services,
+    ]);
+    // Show a success toast
+    toaster.push({
+      message: 'Service was successfully created.',
+      variant: 'success',
+      dismissAfter: 3000,
+    });
+    // Close the modal
+    closeModal();
+  };
+
+  return (
+    <Theme.Provider theme="default">
+      {/* This renders in a portal so it can be anywhere in our JSX */}
+      <Toaster {...toaster} />
+      <Box marginBottom="space60" marginTop={['space10', 'space110']} marginLeft="space60" marginRight="space60">
```

Comment:
Maybe we can add a link to the create pattern from the Paste docs here?
https://paste.twilio.design/patterns/create